editoast/src/views/infra/mod.rs

```diff
@@ -129,6 +130,21 @@ async fn delete(
     .await
 }
 
+/// Update an infra name
+#[put("/<infra>", data = "<new_name>")]
+async fn update_infra_name(
+    infra: i32,
+    new_name: Result<Json<String>, JsonError<'_>>,
```

Comment:
This doesn't match openapi. We're expecting a json like that:

```json
{
    "name": "My new name"
}
```

-----------

You can use (and rename) `CreateInfra` struct.